### PR TITLE
Add Dev Fun

### PR DESCRIPTION
**Pull request by mislavcimpersak**
**https://github.com/mislavcimpersak/awesome-dev-fun**

**This is a list of funny, whimsical and mostly useless scripts/packages from which developers can get a good laugh.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by a7madgamal** [comment]
I'm not a moderator or anything, but I expected 'awesome-fun' to be fun in general not developers kind of fun :D
just a tiny comment

**Comment by mislavcimpersak** [comment]
@a7madgamal ok, I see how it can be a bit misleading :thinking:

@jedahan I'm more inclined to `awesome-dev-fun`.

The general idea is that github is really a dev community so useless libs and such are what constitutes as fun here. What content did you expect to be in a list called `awesome-fun`? :blush:

**Comment by a7madgamal** [comment]
@mislavcimpersak https://github.com/tobiasbueschel/awesome-pokemon :D

**Comment by mislavcimpersak** [comment]
@a7madgamal & @jedahan I've had some offline discussions as well and was persuaded to change the name 😸 

The list is now called `awesome-dev-fun` as suggested by @jedahan.
@jedahan thanks for a good suggestion mate 😉 

I've updated the pull request also.

File: readme.md [hunk 1]
```diff
@@ -504,6 +504,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Free Software](https://github.com/johnjago/awesome-free-software) - Free as in freedom.
 - [Framer](https://github.com/podo/awesome-framer) - Prototyping interactive UI designs.
 - [Markdown](https://github.com/BubuAnabelas/awesome-markdown) - Markup language.
+- [Dev Fun](https://github.com/mislavcimpersak/awesome-dev-fun) - Libs/packages/languages that have no real purpose but to make a developer chuckle.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should include a short description about the list project/theme **(not the list itself)**

**Comment by sindresorhus** [comment]
`merge request` => `pull request`

`Awesome dev fun` => `Awesome Dev Fun`

`js from StackOverflow.` => `JS from Stack Overflow.`

---

> Each link description starts with an uppercase character and ends with a period.

Not followed.

---

Submission: http://www.cornify.com/

File: readme.md (hunk 1 above)
Comment (mislavcimpersak) [inline readme.md]: Ouh, something like `Funny dev projects`?

Comment (sindresorhus) [inline readme.md]: > Funny developer projects.

Comment (mislavcimpersak) [inline readme.md]: updated in https://github.com/sindresorhus/awesome/pull/914/commits/8aa8a748c26e33aa07bc148e5b1f77e551272aa5

**Comment by mislavcimpersak** [comment]
> merge request => pull request
> Awesome dev fun => Awesome Dev Fun

fixed in: https://github.com/mislavcimpersak/awesome-dev-fun/commit/9475503a99bcf5bb83fa536cd27c6fe72ade7b0a

> js from StackOverflow. => JS from Stack Overflow.

fixed in: https://github.com/mislavcimpersak/awesome-dev-fun/commit/7162fb338d5f58b31a8aca0077cd1e9ab7015c65

> Each link description starts with an uppercase character and ends with a period.

fixed in: https://github.com/mislavcimpersak/awesome-dev-fun/commit/46897ae9576e5debe0021d2352095c19601ca8e4 & https://github.com/mislavcimpersak/awesome-dev-fun/commit/1cca6bdba956293ee3240f4ceccfe204561ecd91

**Comment by mislavcimpersak** [comment]
Thanks @sindresorhus for the review and adding this list to the awesome list 🤗

**Pull request merged**